Source/FakeItEasy.Tests/Core/DynamicContainerTests.cs

```diff
@@ -116,9 +116,9 @@ private DynamicContainer CreateContainer()
 
         public class OptionsBuilderForTypeWithDummyFactory : IFakeOptionsBuilder
         {
-            public int Priority
+            public Priority Priority
             {
-                get { return 0; }
+                get { return new Priority(0); }

```

Comment:
I do like `Priority.Default`, and nearly had it.
Will do!

I am not so sure about `Priority.Internal`. Right now, we only use the one value (-1), but who's to say that we won't add others? If we're content for that to be a problem for the future, I can do.